elements/pfe-sass/maps/_colors.scss

```diff
@@ -1,3 +1,5 @@
+@import "~@patternfly/patternfly/sass-utilities/scss-variables";
```

Comment:
@starryeyez024 This is a cool syntax, what does the ~ do?

Reply:
it's from "node-sass-package-importer", which @kylebuch8  found, he might be able to add more detail

Reply:
We don't need a package to do this.  This functionality is built into the sass compiling we're using via the `includePath` option.  I'll add notes in the build file on how to do this.